src/sanitizer.js

```diff
@@ -110,6 +110,9 @@ const WHITELISTED_ATTRS = [
   /* Attributes added for amp-bind */
   // TODO(kmh287): Add more whitelisted attributes for bind?
   'text',
+  /* Attributes for amp-subscriptions */
+  'subscriptions-action',
```

Comment:
Also need `subscriptions-actions` and `subscriptions-display`, right?

Can you please also add a test for this?

Reply:
Yep, Done and added the test